### ISSUE TITLE
ResponsiveBackgrounds.init() not required and options are ignored

### ISSUE THREAD

**Issue by mishterk** [webatvantage/Responsive-Background-Images#1]
G'day,

Is this still being worked on? I definitely need to add something like this to my toolkit.

Anyway, I was just doing some quick testing using the demo provided in the README file, but the plugin works without calling the init() method. I also noticed that any options passed to the init method are just ignored. I was only playing with the interval, but that had no effect on change time.

Cheers

**Comment by robertnurijanyan** [comment]
Hey mishtershmart,

Thanks for the feedback. I've updates the readme to match the current version.
I'll have a look at the parameters as soon as possible. In the mean time you can set the interval directly in the source file (currently on line 47).

```
    // Default settings
    var defaults = {
        envs: ['xs', 'sm', 'md', 'lg'],
        selector: '.bg-responsive',
        interval: 500
    };
```

I'll keep you posted once I fixed it!

**Comment by robertnurijanyan** [comment]
Should be fixed in v1.02
